src/extension.ts

```diff
@@ -107,9 +108,19 @@ async function getLatestRelease(): Promise<{ tagName: string, url: string }> {
     throw new Error(`Received an error code from Github: ${response.status}`);
   }
   const { tag_name, assets } = JSON.parse(response.responseText);
-  const exe = assets.find((asset: any) => asset.name === "redscript-ide.exe");
+  const platform = os.platform();
+  const arch = os.arch();
+  // these are the binaries built in redscript-ide on github
+  const binary = platform === "win32"
+  ? "redscript-ide.exe"
```

Comment:
I think we can use the same convention for windows, and actually, maybe it'd be a good idea to use standard triplet platform format for all of them: https://stackoverflow.com/a/76122210

Reply:
done in 7862ed3 and jac3km4/redscript-ide@094b532